components/src/Box/Box.story.js

```diff
@@ -1,16 +1,14 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
-import { withInfo } from "@storybook/addon-info";
+// import { withInfo } from "@storybook/addon-info";
```

Comment:
this could be removed instead of commented out.

Reply:
Removed.